src/qibolab/tests/conftest.py

```diff
@@ -4,24 +4,20 @@
 
 
 def pytest_addoption(parser):
-    parser.addoption("--platforms", type=str, action="store", default=None, help="qpu platforms to test on")
+    parser.addoption("--platform", type=str, action="store", default=None, help="qpu platform to test")
 
 
 def pytest_configure(config):
     config.addinivalue_line("markers", "qpu: mark tests that require qpu")
 
 
 def pytest_generate_tests(metafunc):
-    platforms = metafunc.config.option.platforms
-    platforms = [] if platforms is None else platforms.split(",")
-
-    if metafunc.module.__name__ == "qibolab.tests.test_instruments_rohde_schwarz":
-        pytest.skip("Skipping Rohde Schwarz tests because it is not available in qpu5q.")
+    platform_name = metafunc.config.option.platform
 
     if "platform_name" in metafunc.fixturenames:
         if "qubit" in metafunc.fixturenames:
             # TODO: Do backend initialization here instead of every test (currently does not work)
-            qubits = [(n, q) for n in platforms for q in range(Platform(n).nqubits)]
+            qubits = [(platform_name, q) for q in range(Platform(platform_name).nqubits)]
```

Comment:
Hi @stavros11,
to support qubits with string IDs, I would use:
```python
qubits = [(platform_name, q) for q in Platform(platform_name).qubits]
```

Reply:
Done, thanks.